lnbits/templates/components.vue

```diff
@@ -1443,7 +1445,7 @@
         </q-tooltip>
       </q-btn>
       <q-btn
-        v-if="authMethods.includes('keycloak-auth')"
+        v-if="LNBITS_AUTH_METHODS.includes('keycloak-auth')"
```

Comment:
why mix globals here?

Reply:
i think i removed the property authMethods, now parsing is done globally, and before that it was done inside the component

Reply:
The question is "why"?
It was better before.

Reply:
why passing around properties when they are static in globals?

Reply:
Not in scope of this PR

Reply:
reverted all refactors from components